src/bulk-load.ts

```diff
@@ -92,6 +92,10 @@ class RowTransform extends Transform {
 
     for (let i = 0; i < this.columns.length; i++) {
       const c = this.columns[i];
+      const error = c.type.validate(row[i]);
+      if (error instanceof TypeError) {
+        throw error;
+      }
```

Comment:
This is a breaking change, because validation for bulk loads has never happened before. I agree that we should make this change, but in a separate PR so we can have the version number bumped.

Reply:
I will make a separate PR for this

Reply:
A new PR is made # 1065. I have reverted changes from this PR.